website/blog/2019-08-31-dandelion-overview.md

```diff
@@ -1,73 +1,73 @@
 
-If you don't want to deploy yourself, you can also [demo the functionality on rinkeby](https://rinkeby.aragon.org/#/tryredemptions/0x18a9713625256548670ad979d51a6b9fad5b6c45) you just need to mint yourself some tokens in the tokens app and then try redeeming them.
+If you don't want to deploy Redemptions yourself you can [demo the functionality on rinkeby](https://rinkeby.aragon.org/#/tryredemptions/0x18a9713625256548670ad979d51a6b9fad5b6c45). You'll then be able to mint yourself some Dandelion Org tokens and deem them for a portion of the DAO's assets.
 
 ### Token Request:
 > Allows users to propose minting tokens in exchange for a payment to the organization, subject to the approval of existing members.
 
-MolochDAO has two proposal types: one where shares are minted in exchange for tribute and another to allocate funds towards proposals. With Dandelion, there are many types of proposals possible depending on what permissions the org has been granted and what apps are installed. With an app like [Aragon Agent](https://twitter.com/lkngtn/status/1147237525150425089) installed, a Dandelion org can have proposals to interact with arbitrary external contracts.
+MolochDAO has two proposal types: one where shares are minted in exchange for tribute and another to allocate funds towards proposals.
 
-However, there is currently no way in Aragon for a user to propose minting tokens in exchange for payment, and this is what the Token Request app provides.
+We created the Token Request app to allow addresses to create a request for a Dandelion Org's membership tokens in exchange for any other ERC-20 tokens. Dandelion Org members can then choose to accept or deny the address's request. Dandelion Orgs will also be able to create proposals to allocate funds via the voting and finance apps. In addition, Dandelion Orgs also allow members to create many more types of proposals depending on how permissions are set and which apps are installed. With an app like [Aragon Agent](https://twitter.com/lkngtn/status/1147237525150425089) Dandelion Orgs can create proposals that interact with any external contract.
 
-We think this will be a huge benefit to the wider Aragon ecosystem even outside of the Dandelion template as it enables a form of permissioned fundraising.
+We anticipate that the Token Request app will be of benefit, not just to Dandelion Orgs, but the wider Aragon ecosystem. One use case would be a form of permissioned fundraising. The most popular method of fundraising in the Ethereum ecosystem is the ICO. This is somewhat similar to an IPO as tokens are free for purchase and trade on open markets. The Token Request app allows for selective and negotiated token sales. This is somewhat similar to the venture model where startups and investors negotiate, agree on a price, and then investors get voting rights and a share of the company in exchange for cash.
```

Comment:
`The most popular method of fundraising in the Ethereum ecosystem is the ICO. This is somewhat similar to an IPO as tokens are free for purchase and trade on open markets.` not sure this tangent adds anything.

Reply:
changed